lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -264,21 +267,27 @@ defmodule ExUnit.Runner do
             end
           end)
 
-        send(parent, {self(), :test_finished, %{test | time: us}})
+        send(parent_pid, {self(), :test_finished, %{test | time: us}})
         exit(:shutdown)
       end)
 
-    timeout = get_timeout(test.tags, config)
-    test = receive_test_reply(test, test_pid, test_ref, timeout)
+    reply_options = %{
+      pid: test_pid,
+      ref: test_ref,
+      parent_pid: parent_pid,
+      timeout: timeout,
+      config: config
+    }
 
-    exec_on_exit(test, test_pid, timeout)
-  end
+    test = receive_test_reply(test, reply_options)
 
-  defp generate_test_seed(%{seed: seed}, %ExUnit.Test{module: module, name: name}) do
-    :rand.seed(@rand_algorithm, {:erlang.phash2(module), :erlang.phash2(name), seed})
+    exec_on_exit(test, test_pid, timeout)
   end
 
-  defp receive_test_reply(test, test_pid, test_ref, timeout) do
+  defp receive_test_reply(
+         test,
+         %{pid: test_pid, ref: test_ref, timeout: timeout} = options
```

Comment:
Please move the pattern to inside the function.

Reply:
done